lib/commands.js

```diff
@@ -125,6 +126,7 @@ module.exports = function(config) {
       operations.confirmTemplate,
       operations.mergeMetadata,
       operations.saveTemplate,
+      operations.saveConfig,
```

Comment:
What if you provided an invalid parameter set and the update rolls back? I think this would need to happen _after_ a successful deployment.